packages/communication-react/CHANGELOG.stable.md

```diff
@@ -10,46 +10,40 @@ Mon, 08 Apr 2024 20:34:39 GMT
 [Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.15.0-beta.1...@azure/communication-react_v1.15.0)
```

Comment:
Should edit this to line to:
`[Compare changes](https://github.com/azure/communication-ui-library/compare/1.14.0...1.15.0)`
and the edit above link to:
`## [1.15.0](https://github.com/azure/communication-ui-library/tree/1.15.0)`